subprojects/docs/src/docs/userguide/organizingGradleProjects.adoc

```diff
@@ -117,13 +156,54 @@ A typical project including `buildSrc` has the following layout.
 
 In Gradle, properties can be define in the build script, in a `gradle.properties` file or as parameters on the command line.
 
-It's common to declare properties on the command line for ad-hoc scenarios. For example you may want pass in a specific property value to control runtime behavior just for this one invocation of the build. Properties in a build script can easily become a maintenance headache and convolute the build script logic. The `gradle.properties` helps with keeping properties separate from the build script and should be explored as viable option. It's a good location for placing <<sec:gradle_configuration_properties,properties that control the build environment>>.
+It's common to declare properties on the command line for ad-hoc scenarios.
+For example you may want pass in a specific property value to control runtime behavior just for this one invocation of the build.
+Properties in a build script can easily become a maintenance headache and convolute the build script logic.
+The `gradle.properties` helps with keeping properties separate from the build script and should be explored as viable option.
+It's a good location for placing <<sec:gradle_configuration_properties,properties that control the build environment>>.
 
-A typical project setup places the `gradle.properties` file in the root directory of the build. Alternatively, the file can also live in the `GRADLE_USER_HOME` directory if you want to it apply to all builds on your machine.
+A typical project setup places the `gradle.properties` file in the root directory of the build.
+Alternatively, the file can also live in the `GRADLE_USER_HOME` directory if you want to it apply to all builds on your machine.
 
 ----
 .
 ├── build.gradle
 ├── gradle.properties
 └── settings.gradle
 ----
+
+=== Avoid writing to the same output directory from different tasks
+
+Tasks should define inputs and outputs to enable proper functioning of <<sec:up_to_date_checks,incremental build functionality>>.
+When declaring the outputs of a task, make sure that the directory for writing outputs is unique among all the tasks in your project.
+
+Intermingling or overwriting output files produced by different tasks compromises up-to-date checking causing slower builds.
+In turn, these filesystem changes may prevent Gradle's <<build_cache,build cache>> from properly identifying and caching what would otherwise be cacheable tasks.
+
+[[sec:custom_gradle_distribution]]
+=== Standardizing builds with a custom Gradle distribution
+
+<<init_scripts,Initialization scripts>> make it extremely easy to apply build logic across all projects on a single machine.
+Specifying an in-house binary repository and its credentials is a typical use case for setting up an initialization script.
+
+Often times enterprises want to standardize the build platform for all projects in the organization by defining common conventions or rules.
```

Comment:
This is a great opening paragraph. Move it above? 

⭕️ can omit "times"

Reply:
done